pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[build-system]
+requires = ["scikit-build-core", "pybind11"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "desbordante"
+version = "1.0.0"
+description = "Python bindings for Desbordante: https://github.com/Mstrutov/Desbordante"
```

Comment:
> `[project.urls]` seems like a right section for this GitHub link (and maybe for the website link)
This will be specified in README_PYPI in a separated pull request with all metadata for pip package because of a variety of options.